webapp/src/mutator.ts

```diff
@@ -436,7 +436,7 @@ class Mutator {
                         let option = newTemplate.options.find((o: IPropertyOption) => o.value === oldValue)
                         if (!option) {
                             option = {
-                                id: Utils.createGuid(),
+                                id: Utils.createGuid(''),
```

Comment:
same here, I think this is a `block`

Reply:
Fixed via https://github.com/mattermost/focalboard/pull/1369/commits/03daaf722a0bff201708cf27b0434c8fbfef4f4c